modules/vector-search/pages/run-vector-search-ui.adoc

```diff
@@ -0,0 +1,43 @@
+= Run A Vector Search with the Capella UI 
+:page-topic-type: guide
+:description: Run a Vector Search query from the Couchbase Capella UI to preview the search results from a Vector Search index.
+
+[abstract]
+{description}
+
+== Prerequisites 
+
+* You have the Search Service enabled on a node in your database. 
+
+* You have created a Vector Search index. 
++
+For more information about how to create a Vector Search index, see xref:create-vector-search-index-ui.adoc[].
+
+* You have logged in to the Couchbase Capella UI. 
+
+== Procedure 
+
+To run a Vector Search with the Capella UI: 
+
+. On the *Databases* page, select the database where you created your Search index. 
+. Go to menu:Data Tools[Search]. 
+. Next to your Vector Search index, click btn:[Search].
+. In the *Search* field, enter a search query. 
++
+For example, the following query searches for the top 2 vectors similar to the vector `[ 0.707106781186548, 0, 0.707106781186548 ]` in the `colorvect_dot` field:
++
+[source,json]
+----
+include::example$run-vector-search-payload-ui.jsonc[]
+----
```

Comment:
Because it is very different from all other queries in Capella. This isn't based on SQL++ at all - it's full text search. 

This also isn't the result, but the actual query you need to put into the Search field to return a result.